### DISCUSSION TITLE
A contributors repository for the examples?

### DISCUSSION THREAD

**Discussion by D8H** [4ian/GDevelop#2947] [Enhancement ideas (and their technical discussions)]
It's hard to work collaboratively on PR of the official repository.
Do you think that a secondary repository with wider accesses could smooth the process?
People could commit directly on a common branch and PR to the official repository from this branch.

**Comment by Bouh** [comment]
This will multiply the load of the GDevelop team, if it is maintained and used by the community personally I agree. I would only look at the final reviews and submissions on the official repo.
From my point of view it's mostly an arrangement between the contributors, who will then submit the output.

**Comment by Midhil457** [comment]
I agree with having one. It will encourage contributing to examples. Repos should be only made for ones which need them for proper development of large and important ones like the platformer, shootaila, space shooter games, and many other. The repos could be under the GDevelopApp organization. Quick fixes can be easily done. Currently for the active development of Minesweeper, the contributions are separate  zips and I don't know how they will be merged. The creator of that specific example could have write access to the respective repos. this could lead to continuous updation of examples and can increase the quality of examples too

**Reply by D8H** [reply]
I think that only one community fork could be enough. People could create one branch for each PR to make to the official repository.
The fork could be configured to have many contributors:
https://docs.github.com/en/github/setting-up-and-managing-your-github-user-account/managing-user-account-settings/permission-levels-for-a-user-account-repository
It's not safe, but everyone can have a copy of their work on their own fork and locally.

It could be managed by someone of the community and ownership can be passed if the person is no longer interested to handle right accesses.

**Reply by Bouh** [reply]
> Currently for the active development of Minesweeper, the contributions are separate zips and I don't know how they will be merged.

I take care of mergings ;)
